doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -332,6 +332,28 @@ In case of custom format datetimes in filename, an additional *format* element s
 
     regex=.*([0-9]{10}).*,format=yyyyMMddHH
 
+In case of reduced precision of temporal information, where there is the need to get the higher time included in that reduced value, an additional *,inclusive=true* element should be added.
```

Comment:
``inclusive`` seems to suggest that something that was left out is not added, but instead, this is a swap, getting the high time instead of the low time in a low precision date parse. I'd rather call it ``useHighTime`` instead (or anything else that is clear about the second time being used).

Reply:
About the docs, I noticed that GeoTools links in the GeoServer docs, but the link is dangling:
https://docs.geotools.org/latest/userguide/library/coverage/mosaic.html
points to http://docs.geoserver.org/stable/en/user/tutorials/image_mosaic_plugin/imagemosaic.html which does not exist anymore... probably best to update the links pointing both at the reference documentation and tutorials, in the associated GeoTools PR.

Reply:
I'm going to update it accordingly